tests/Microsoft.Bot.Builder.TemplateManager/TemplateManagerTests.cs

```diff
@@ -1,120 +1,90 @@
 
-        public TestContext TestContext { get; set; }
-
-        [AssemblyInitialize]
-        public static void SetupDictionaries(TestContext testContext)
+        public TemplateManagerTests(TemplateFixture templateFixture)
         {
-            _testContext = testContext;
-            templates1 = new LanguageTemplateDictionary
-            {
-                ["default"] = new TemplateIdMap
-                {
-                    { "stringTemplate", (context, data) => $"default: {data.name}" },
-                    { "activityTemplate", (context, data) => { return new Activity() { Type = ActivityTypes.Message, Text = $"(Activity)default: {data.name}" }; } },
-                    { "stringTemplate2", (context, data) => $"default: Yo {data.name}" },
-                },
-                ["en"] = new TemplateIdMap
-                {
-                    { "stringTemplate", (context, data) => $"en: {data.name}" },
-                    { "activityTemplate", (context, data) => { return new Activity() { Type = ActivityTypes.Message, Text = $"(Activity)en: {data.name}" }; } },
-                    { "stringTemplate2", (context, data) => $"en: Yo {data.name}" },
-                },
-                ["fr"] = new TemplateIdMap
-                {
-                    { "stringTemplate", (context, data) => $"fr: {data.name}" },
-                    { "activityTemplate", (context, data) => { return new Activity() { Type = ActivityTypes.Message, Text = $"(Activity)fr: {data.name}" }; } },
-                    { "stringTemplate2", (context, data) => $"fr: Yo {data.name}" },
-                },
-            };
-            templates2 = new LanguageTemplateDictionary
-            {
-                ["en"] = new TemplateIdMap
-                {
-                    { "stringTemplate2", (context, data) => $"en: StringTemplate2 override {data.name}" },
-                },
-            };
+            this.templateFixture = templateFixture;
         }
 
-        [TestMethod]
+        [Fact]
         public void TemplateManager_Registration()
         {
             var templateManager = new TemplateManager();
-            Assert.AreEqual(templateManager.List().Count, 0, "nothing registered yet");
-            var templateEngine1 = new DictionaryRenderer(templates1);
-            var templateEngine2 = new DictionaryRenderer(templates2);
+            Assert.Empty(templateManager.List());
+
+            var templateEngine1 = new DictionaryRenderer(templateFixture.Templates1);
+            var templateEngine2 = new DictionaryRenderer(templateFixture.Templates2);
             templateManager.Register(templateEngine1);
-            Assert.AreEqual(templateManager.List().Count, 1, "one registered");
+            Assert.Single(templateManager.List());
 
             templateManager.Register(templateEngine1);
-            Assert.AreEqual(templateManager.List().Count, 1, "only  one registered");
+            Assert.Equal(1, templateManager.List().Count);
```

Comment:
Can we use `single` in here?
```suggestion
            Assert.Single(templateManager.List());
```